.github/workflows/prepare-action/action.yml

```
@@ -0,0 +1,30 @@
+# SPDX-FileCopyrightText: 2022 Helmholtz Centre for Environmental Research (UFZ)
+# SPDX-FileCopyrightText: 2022 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
+#
+# SPDX-License-Identifier: Apache-2.0
+
+---
+name: Install dependencies and prepare the environment.
+description: Install the necessary dependencies for jobs.
+runs:
+  using: composite
+  steps:
+    - name: Set up Python 3.
+      uses: actions/setup-python@v3
+      id: setup-python
+      with:
+        python-version: '3.9'
```

According to https://github.com/actions/setup-python#caching-packages-dependencies

```suggestion
        python-version: '3.9'
        cache: 'pipenv'
```